hhs_hosp/delphi_hhs/run.py

```diff
@@ -78,18 +78,41 @@ def run_module():
             raise Exception(f"Bad result from Epidata: {response['message']}")
         dfs.append(pd.DataFrame(response['epidata']))
     all_columns = pd.concat(dfs)
-    signal = pd.DataFrame({
-        "geo_id": all_columns.state.apply(str.lower),
-        "timestamp":int_date_to_previous_day_datetime(all_columns.date),
-        "val": \
-        all_columns.previous_day_admission_adult_covid_confirmed + \
-        all_columns.previous_day_admission_pediatric_covid_confirmed,
-        "se": None,
-        "sample_size": None
-    })
-    create_export_csv(
-        signal,
-        params["export_dir"],
-        "state",
-        HOSPITALIZATIONS
+
+    for sig in SIGNALS:
+        create_export_csv(
+            make_signal(all_columns, sig),
+            params["export_dir"],
+            "state",
+            sig
         )
+
+def make_signal(all_columns, sig):
+    """Generate column sums according to signal name."""
+    assert sig in SIGNALS, f"Unexpected signal name '{sig}';" + \
+        " familiar names are '{', '.join(SIGNALS)}'"
+    if sig == CONFIRMED:
+        return pd.DataFrame({
+            "geo_id": all_columns.state.apply(str.lower),
+            "timestamp":int_date_to_previous_day_datetime(all_columns.date),
+            "val": \
+            all_columns.previous_day_admission_adult_covid_confirmed + \
+            all_columns.previous_day_admission_pediatric_covid_confirmed,
+            "se": None,
+            "sample_size": None
+        })
+    if sig == SUM_CONF_SUSP:
+        return pd.DataFrame({
+            "geo_id": all_columns.state.apply(str.lower),
+            "timestamp":int_date_to_previous_day_datetime(all_columns.date),
+            "val": \
+            all_columns.previous_day_admission_adult_covid_confirmed + \
+            all_columns.previous_day_admission_adult_covid_suspected + \
+            all_columns.previous_day_admission_pediatric_covid_confirmed + \
+            all_columns.previous_day_admission_pediatric_covid_suspected,
+            "se": None,
+            "sample_size": None
+        })
+    raise Exception(
+        "Bad programmer: signal '{sig}' in SIGNALS but not handled in make_signal"
```

Comment:
:laughing: 

related discussion for the future: should we add some custom exceptions that all the indicators can share? Not sure how many cases there would be and don't think it's that important right now, but might be a nice-to-have.